src/seedu/addressbook/data/person/Person.java

```diff
@@ -59,6 +64,13 @@ public UniqueTagList getTags() {
         return new UniqueTagList(tags);
     }
 
+    /**
+     * Updates the sequence number to count the next person
+     */
+    public void updateSequenceNumber() {
```

Comment:
Is there really a need to make this public? we should only allow the sequenceNumber to be incremented after a new person is successfully added right?

Reply:
Ah yes, that's right. Private would be a better choice for the method to update the class variable in this case. Thanks!